app/resources/repo-menu.html.template

```diff
@@ -19,4 +19,6 @@
       {% if option.test_mode %}({% trans "in test mode" %}){% endif %}</a><br>
   {% endfor %}
   </ul>
+{% else %}
```

Comment:
Can you revert this part? I don't think we need this because we also have a header.